sktime/detection/hmm.py

```diff
@@ -376,6 +381,16 @@ def _fit(self, X, Y=None):
         self :
             Reference to self.
         """
+        X = X.values.flatten()
+        self.num_obs = len(X)
+        emi_probs = self._make_emission_probs(self.emission_funcs, X)
+        self.trans_prob, self.trans_id = self._calculate_trans_mats(
```

Comment:
excellent idea! Though convention for fitted parms is that the shoud *end* with underscore, so `self.trans_prob_` etc